### DISCUSSION TITLE
Why not use hour angles in Stellarium's equatorial grid?

### DISCUSSION THREAD

**Discussion by MetricRocketeer** [Stellarium/stellarium#3446] [General]
Hi Stellarium colleagues,

I just recently downloaded Stellarium, and it is great!

However, I have this question please.  In the equatorial grid, degrees are used for the East-West coordinates.  But shouldn't hour angles be used instead?

Thank you.

Stanley

**Comment by gzotti** [comment]
There are fixed (hour angle) and mobile (right ascension) grids available. Whether it's counted in degrees or hours depends on further settings.

**Reply by MetricRocketeer** [reply]
Hi @gzotti or anyone else,

Thank you for your reply.

Could you please tell me where I find that setting to change the grid.

Stanley
